Snakefile

```diff
@@ -203,3 +211,27 @@ rule bam_to_fastq:
         " I={input.bam}"
         " FASTQ={output.r1_fastq}"
         " SECOND_END_FASTQ={output.r2_fastq} 2>> {log}"
+
+if config['call_variants']:
+    rule call_variants_freebayes:
+        output:
+             vcf = "{dir}/reference.vcf"
+        input:
+             bam = "{dir}/mapped.sorted.tag.mkdup.bcmerge.filt.bam"
+        log: "{dir}/call_variants_freebayes.log"
+        params:
+            reference = config["bowtie2_reference"] + ".fasta" # I am unsure if this is a good solution, but it works.
+        shell:
+             "freebayes"
+             " -f {params.reference}"
+             " {input.bam} 1> {output.vcf} 2> {log}"
+elif os.path.exists(config['reference_variants']):
```

Comment:
If a path is supplied but is given erroneously the `elif` statement the error message would be confounding since it also checks if path exists. I think a better solution is to only check for the argument and letting snakemake point out any non-existing paths (as we would for e.g. missing read files).

```suggestion
elif config['reference_variants']:
```